### PR TITLE
Load test for exporting keys at scale

### PR DESCRIPTION
**Pull request by chaodaiG**
- Add a new test replacing the old load test for cleaning up, this new test measures performance for both export and cleanup
- Implemented a slightly better way of generating exports
- There is still a workaround of generating exports, which is also hard to be done using `/publish` endpoint for scaled workloads. So it's still blocked from being merged with integration test

The test runs 660 seconds on my development machine, took 2.2 minutes export and 265 milliseconds cleaning up.

The mako set up will be added in a separate PR

### CODE REVIEW

File: internal/performance/export_test.go [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+		if err != nil {
+			t.Fatal(err)
+		}
+		if updated != keysPerPublish {
+			t.Fatalf("Want updated: %d, got %d", keysPerPublish, updated)
+		}
+	}
+
+	// Ensure all keys published
+	integration.Eventually(t, 30, func() error {
```
Comment (mikehelmick) [inline internal/performance/export_test.go:146]: is this step needed? doesn't finding all the keys in and export tell us that?

Comment (chaodaiG) [reply internal/performance/export_test.go]: Good point, it's not needed. Deleted

**Pull request merged**